cpp/src/Ice/PropertyNames.h

```diff
@@ -11,18 +11,23 @@
 
 namespace IceInternal
 {
+    struct PropertyArray;
     struct Property
     {
         const char* pattern;
-        bool usesRegex;
         const char* defaultValue;
+        bool usesRegex;
         bool deprecated;
+        const PropertyArray* propertyClass;
+        bool prefixOnly;
 
-        Property(const char* n, bool r, const char* dv, bool d) :
+        Property(const char* n, const char* dv, bool r, bool d, const PropertyArray* pc, bool po) :
```

Comment:
I find parameters that shadow the field names more readable.

Reply:
I removed the constructors entirely as this is a struct.